classes/genre.rb

```diff
@@ -0,0 +1,14 @@
+class Genre
+  attr_accessor :name, :items
+
+  def initialize(name)
+    @id = Random.rand(1..1000)
+    @name = name
+    @items = []
+  end
+
+  def add_genre(_genre)
+    @item = item
+    item.genre.push(self) unless item.include?(self)
+  end
```

Comment:
I think the project requires we use the  add_item method in the genre class and push self into @items just like this

```
def add_item(item)
    @items.push(item) unless @items.include?(item)
    item.add_genre(self)
  end
```
## Project requirement
![genre](https://user-images.githubusercontent.com/99246359/189965394-b2492378-2b92-4a9e-9ad2-83ab3c2eec1a.png)

What do you think about this? 
I think you can also `require_relative 'item'` in your genre.rb file

Reply:
Noted